pkgs/development/tools/yourkit-java/forbid-desktop-item-creation

```diff
@@ -0,0 +1,16 @@
+CONFIG_DIR=$HOME/.yjp/config
+if [ ! -d $CONFIG_DIR ]; then
+    mkdir -p $CONFIG_DIR
+fi
```

Comment:
Do you think it’d be possible to do this after assignment of `YS` in the startup script so that changing `YOURKIT_HOME` is handled?